aws/rust-runtime/aws-http/Cargo.toml

```diff
@@ -28,7 +28,6 @@ aws-smithy-checksums = { path = "../../../rust-runtime/aws-smithy-checksums" }
 aws-smithy-protocol-test = { path = "../../../rust-runtime/aws-smithy-protocol-test" }
 bytes-utils = "0.1.2"
 env_logger = "0.9"
-http = "0.2.3"
```

Comment:
I noticed that this was in both the regular and dev dependencies so I removed it from dev dependencies.